### DISCUSSION TITLE
bot inventory

### DISCUSSION THREAD

**Discussion by AshDoesStuf** [PrismarineJS/mineflayer#2633] [Q&A]
when the bots inventory is full, how do i make so that it look for the nearest chest and deposits its entire inventory?

**Comment by amoraschi** [comment]
https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#botfindblockoptions
To find a chest

https://github.com/PrismarineJS/mineflayer-pathfinder
To go to the chest

https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#botopencontainercontainerblock-or-containerentity
To open the chest

https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#windowdeposititemtype-metadata-count-nbt
To deposit the items

https://github.com/PrismarineJS/mineflayer/blob/master/docs/api.md#windowclose
To close the chest

**Reply by AshDoesStuf** [reply]
thx
